include/asterisk/channel.h

```diff
@@ -3076,14 +3082,49 @@ struct ast_channel *ast_channel_iterator_next(struct ast_channel_iterator *i);
  *
  * \details
  * This function executes a callback one time for each active channel on the
- * system.  The channel is provided as an argument to the function.
+ * system.  The channel (or thing) to be searched for is provided as an argument (arg)
+ * to the function.
  *
+ * \param arg Must be a char *, Typically either a channel name or a channel uniqueid.
+ *   In some cases exten@context is used.
+ *   Will be passed to cb_fn as its arg param
+ * \param data Arbitrary data to be passed to cb_fn as its data param
+ *
+ * \note This function SHOULD NOT be called unlessfully understanding the implications.
```

Comment:
unlessfully?

Reply:
Haha... yeah I noticed that after I sent the last push.  Should be 'unless fully'